### PR TITLE
E2e

### PR DESCRIPTION
**Pull request by Dwaynekj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,17 +3,29 @@ Demo of media pipe python integration with Django
...
+  - `root@bca08b91ff63:/mediapipe# GLOG_logtostderr=1 bazel run --define MEDIAPIPE_DISABLE_GPU=1 mediapipe/examples/desktop/hello_world:hello_world`
+
+#### Demo Server Container
+- `docker-compose up`
+- `curl -d '{"archive_id": "013bc8ff-f069-44d9-af77-d170acd1c9c6"}' -H 'Content-Type: application/json' localhost:8080/hand`
+
+### Cloud Run
+- ...
+
+### Temporary GCP Server
```
Comment (ninalutz) [inline README.md:42]: @Dwaynekj are you able to add the instructions for our group account yet or is that still being set up?

File: env-sample [hunk 2]
```diff
@@ -0,0 +1,4 @@
+AWS_SECRET_ACCESS_KEY=
```
Comment (ninalutz) [inline env-sample:1]: Are these supposed to be filled out?

File: README.md [hunk 3]
```diff
@@ -3,17 +3,29 @@ Demo of media pipe python integration with Django
...
-- Temporary Server (Dwayne's Personal) `gcloud beta compute ssh --zone "us-east4-c" "mediapipe-server" --project "mystical-slate-241320"`
+### Docker / Local
+
+#### Mediapipe Container
+- Taken from https://google.github.io/mediapipe/getting_started/install.html
+- In a sibling folder, `git clone git@github.com:mitmedialab/mediapipe.git`
+- `docker build --tag=mediapipe .`
+- Quick Test (optiona1)
+  - `docker run -it --name mediapipe mediapipe:latest`
+  - `root@bca08b91ff63:/mediapipe# GLOG_logtostderr=1 bazel run --define MEDIAPIPE_DISABLE_GPU=1 mediapipe/examples/desktop/hello_world:hello_world`
```
Comment (ninalutz) [inline README.md]: I slacked you but unfortunately having trouble running these commands :( So sorry!

File: README.md [hunk 4]
```diff
@@ -3,17 +3,29 @@ Demo of media pipe python integration with Django
...
   - and https://dev.classmethod.jp/articles/mediapipe-install-on-aws-ec2-with-gpu-english/
 - `GLOG_logtostderr=1 bazel run --define MEDIAPIPE_DISABLE_GPU=1 mediapipe/examples/desktop/hello_world:hello_world`
 
-### Docker / Local
-- Taken from https://google.github.io/mediapipe/getting_started/install.html
-- `docker build --tag=mediapipe .`
-- `docker run -it --name mediapipe mediapipe:latest`
-- `root@bca08b91ff63:/mediapipe# GLOG_logtostderr=1 bazel run --define MEDIAPIPE_DISABLE_GPU=1 mediapipe/examples/desktop/hello_world:hello_world`
 
 ## Test Data
```
Comment (ninalutz) [inline README.md:49]: Thank you so much for including my test links!!! :D -- I'll add the full raw video in a PR tho too! :)

File: README.md [hunk 5]
```diff
@@ -3,17 +3,29 @@ Demo of media pipe python integration with Django
...
+- `docker build --tag=mediapipe .`
+- Quick Test (optiona1)
+  - `docker run -it --name mediapipe mediapipe:latest`
+  - `root@bca08b91ff63:/mediapipe# GLOG_logtostderr=1 bazel run --define MEDIAPIPE_DISABLE_GPU=1 mediapipe/examples/desktop/hello_world:hello_world`
+
+#### Demo Server Container
+- `docker-compose up`
+- `curl -d '{"archive_id": "013bc8ff-f069-44d9-af77-d170acd1c9c6"}' -H 'Content-Type: application/json' localhost:8080/hand`
+
+### Cloud Run
```
Comment (ninalutz) [inline README.md:39]: Will this be replaced with the group's GCP?

**Review by ninalutz — CHANGES_REQUESTED** [review]
Added some comments :) Thanks so much Dwayne! :)

File: README.md (hunk 1 above)
Comment (Dwaynekj) [reply README.md:42]: Can't add them yet. I need Emily to add the credit card otherwise we can't use these resources

File: env-sample (hunk 2 above)
Comment (Dwaynekj) [reply env-sample:1]: this is the example file. The real one is in the gitignore for security

File: README.md (hunk 4 above)
Comment (Dwaynekj) [reply README.md:49]: thanks!
yea send me a full example video and I can do more testing

File: README.md (hunk 5 above)
Comment (Dwaynekj) [reply README.md:39]: Ekene and I are still discussing how he wants to handle that. I think he is leaning to using our media lab emails and abandoning poeticjustice.group

**Review by ninalutz — APPROVED** [review]

**Pull request merged**